airflow/providers/apache/cassandra/README.md

```diff
@@ -0,0 +1,105 @@
+## Backport package
+
+This is a backport providers package for `apache.cassandra` provider. All classes for this provider package
+are in `airflow.providers.apache.cassandra` python package.
+
+## Installation
+
+You can install this package on top of an existing airflow 1.10.* installation via
+`pip install apache-airflow-backport-providers-apache-cassandra`
+
+## Compatibility
+
+For full compatibility and test status of the backport packages check
+[Airflow Backport Package Compatibility](https://cwiki.apache.org/confluence/display/AIRFLOW/Backported+providers+packages+for+Airflow+1.10.*+series)
+
+## PIP requirements
+
+| PIP package      | Version required   |
+|:-----------------|:-------------------|
+| cassandra-driver | &gt;=3.13.0,&lt;3.21.0   |
+
+# Provider class summary
+
+All classes in Airflow 2.0 are in `airflow.providers.apache.cassandra` package.
+
+
+
+
+## Sensors
+
+
+
+### Moved sensors
+
+| Airflow 2.0 sensors: `airflow.providers.apache.cassandra` package                                                                          | Airflow 1.10.* previous location (usually `airflow.contrib`)                                                                                                    |
+|:-------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [sensors.record.CassandraRecordSensor](https://github.com/apache/airflow/blob/master/airflow/providers/apache/cassandra/sensors/record.py) | [sensors.cassandra_record_sensor.CassandraRecordSensor](https://github.com/apache/airflow/blob/v1-10-stable/airflow/contrib/sensors/cassandra_record_sensor.py) |
+| [sensors.table.CassandraTableSensor](https://github.com/apache/airflow/blob/master/airflow/providers/apache/cassandra/sensors/table.py)    | [sensors.cassandra_table_sensor.CassandraTableSensor](https://github.com/apache/airflow/blob/v1-10-stable/airflow/contrib/sensors/cassandra_table_sensor.py)    |
+
+
+
+## Hooks
+
+
+
+### Moved hooks
+
+| Airflow 2.0 hooks: `airflow.providers.apache.cassandra` package                                                                      | Airflow 1.10.* previous location (usually `airflow.contrib`)                                                                      |
+|:-------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------|
+| [hooks.cassandra.CassandraHook](https://github.com/apache/airflow/blob/master/airflow/providers/apache/cassandra/hooks/cassandra.py) | [hooks.cassandra_hook.CassandraHook](https://github.com/apache/airflow/blob/v1-10-stable/airflow/contrib/hooks/cassandra_hook.py) |
```

Comment:
![image](https://user-images.githubusercontent.com/34150/82036291-a4111600-9698-11ea-811d-ab7f8ca480ca.png)

So it's not 100% clear what the old class name is here.

I think we should either include the full class name (`airflow.contrib.hooks.cassandra_hook.CassandraHook`)

or at least include the `contrib.` prefix, so that the values in the old class name are all prefixable with `airflow.`

Reply:
TBF in this case it's fine, everything is under contrib, but for the AWS one it's much harder to know guess

Is it easy to _detect_ this, and change the behaviour?

- if it's all under contrib, have the heading say "previous location (`airflow.contrib.*`)"
- if it's mixed then have the header say `airflow.*` and include the `contrib.` in the table cells where needed?

Reply:
I wanted to make the tables "smaller" so I am removing the most common prefixes (table header is (mostly in "airflow.contrib" package) + we have link to the classes themselves. I think - especially in Github which has limited width this makes it much better (otherwise the tables will be very wide and you need to scroll them.

Do you thin @ashb it's too confusing?

Reply:
I think when the prefix is common that makes sense, but adding `contrib.` when it's otherwise ambigious (i.e. AWS case) I think would be good. The columns are still quite narrow so we've got space.

Reply:
Oh hmm, but AWS is very wide already.

Reply:
Yes. confirmed. just removing airflow is much clearer.